CRM/Contribute/BAO/Contribution.php

```diff
@@ -825,15 +838,6 @@ public static function &exportableFields($checkPermission = TRUE) {
         ),
       );
 
-      $contributionRecurId = array(
```

Comment:
As part of this PR I declared this field as exportable so it no longer needs manual adding to exportable fields